base/kustomization.yaml

```diff
@@ -5,6 +5,7 @@ resources:
   - ./thanos-rule
   - ./thanos-compact
   - ./thanos-store
+  - ./thanos-receive
```

Comment:
All looks good to me apart from this, I don't think it should be in the default base. I don't think majority of Thanos deployments need it?